sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -426,7 +426,8 @@ systemFunctionName
 
 singleFieldRelevanceFunctionName
     : MATCH | MATCH_PHRASE | MATCHPHRASE
-    | MATCH_BOOL_PREFIX | MATCH_PHRASE_PREFIX
+    | MATCHPHRASEQUERY | MATCH_BOOL_PREFIX
```

Comment:
nit: add it to the previous line with `MATCH_PHRASE | MATCHPHRASE`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/165/commits/d04916bfbe558347bec18a9b61b3f13697ce270d